src/SolutionTemplate/UnoSolutionTemplate/Shared/Assets/SharedAssets.md

```diff
@@ -0,0 +1,34 @@
+See documentation about assets here : https://github.com/unoplatform/uno/blob/master/doc/articles/features/working-with-assets.md
+
+# Here is a cheat sheet:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5028313)